### PR TITLE
Add installation pipeline template

### PR DESCRIPTION
**Pull request by trepel**
*Motivation*

To have the pipeline configuration versioned and to have an easy way to create the pipeline on different jenkins instances. The pipeline according to this definition is already created here:
https://integreatly-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/all/job/installation-pipeline/

*To Test*
execute this:

```
jenkins-jobs --conf jenkins_jobs.ini test jobs/templates/installation-pipeline.yaml
```

Optionally you can also execute 'update' and run against QE Pony cluster, input parameters can be found here:
https://integreatly-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/installation-pipeline-bastion/18/parameters/

### CODE REVIEW

**Comment by trepel** [comment]
@tremes could you please have a look - for some reason I cannot assign you for review

File: jobs/templates/installation-pipeline.yaml [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+      import jenkins.model.*
+      import hudson.slaves.*
+      import hudson.slaves.EnvironmentVariablesNodeProperty.Entry
+
+      import hudson.plugins.sshslaves.verifiers.*
+
+      String bastionLabel = "${BASTION_URL}-slave"
+
+      timeout(60) {
+          node('cirhos_rhel7') {
```
Comment (tremes) [inline jobs/templates/installation-pipeline.yaml]: Shouldn't we parametrize this as well? WDYT?

**Comment by tremes** [comment]
@trepel Added one minor comment. Otherwise it looks good. Well I am not a member of Integr8ly group so  that's the reason why I can't be assigned for a review.

File: jobs/templates/installation-pipeline.yaml (hunk 1 above)
Comment (tremes) [reply jobs/templates/installation-pipeline.yaml]: Well we can have differrent machines and labels than only `cirhos_rhel7` or not?

Comment (trepel) [reply jobs/templates/installation-pipeline.yaml]: Yes, but such slaves will likely serve for specific purposes only. For instance we have various bastion slaves but such slaves you want only use for the pipeline that deals with specific cluster behind that specific bastion. Using 'cirhos_rhel7' is nice limitation to any non-specific slave.

Can you think of any specific use case when one would want to run this pipeline on specific slave?

Comment (tremes) [reply jobs/templates/installation-pipeline.yaml]: Well I think I can't...I just hope nobody will change our jenkins labels or something like that.:-)

**Pull request merged**